test/onnx/test_verify.py

```diff
@@ -64,21 +64,6 @@ def forward(self, x):
         with self.assertRaisesRegex(RuntimeError, "state_dict changed"):
             verify(MyModel(), x, backend)
 
-    def test_modifying_params(self):
```

Comment:
why is this test case removed?

Reply:
this test case is meant to verify that unused initializers are not being removed when constant_fold is set to False and with this pass is no longer the case